src/main/kotlin/no/nav/familie/tilbake/api/DokumentController.kt

```diff
@@ -50,16 +60,6 @@ class DokumentController(private val varselbrevService: VarselbrevService,
         return varselbrevService.hentForhåndsvisningVarselbrev(forhåndsvisVarselbrevRequest)
     }
 
-    @GetMapping("/forhandsvis-innhentbrev/{behandlingId}",
-                produces = [MediaType.APPLICATION_PDF_VALUE])
-    @Rolletilgangssjekk(minimumBehandlerrolle = Behandlerrolle.SAKSBEHANDLER,
-                        handling = "Forhåndsviser brev",
-                        henteParam = "behandlingId")
-    fun hentForhåndsvisningInnhentDokumentasjonsbrev(@PathVariable behandlingId: UUID,
-                                                     fritekst: String): ByteArray {
-        return innhentDokumentasjonbrevService.hentForhåndsvisningInnhentDokumentasjonBrev(behandlingId, fritekst)
-    }
-
```

Comment:
Er ikke dette endepunktet i bruk av ba-sak?

Reply:
Nei. Det er internt endepunkt for familie-tilbake.
Fagsystemene skal forhåndsvise varselbrev, med endepunktet:
hentForhåndsvisningVarselbrev(@Valid @RequestBody forhåndsvisVarselbrevRequest: ForhåndsvisVarselbrevRequest

Reply:
Jeg leste litt fort, trodde det sto varselbrev på denne.